Tests/SonarScanner.MSBuild.PreProcessor.Test/ArgumentProcessorTests.cs

```diff
@@ -374,8 +374,7 @@ public void PreArgProc_Arguments_Duplicates_WithDifferentFlagsPrefixes()
                     "/d:unique=value5");
 
             // Assert
-            logger.AssertSingleErrorExists("-version:1.2", "1.2");
-            logger.AssertErrorsLogged(1);
+            logger.AssertErrorsLogged(3);
```

Comment:
IMO it is also worth asserting the 3 errors individually with `logger.AssertSingleErrorExists`

Reply:
Yes, I thought `AssertSingleErrorExists` would assert the following two conditions in logical and:
- a single error exists in the logger
- that error contains the provided strings

Instead, the method asserts that a single error exists in the logger, that contains the provided strings.
I added the three checks.